sdk/src/main/scala/io/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -125,6 +140,23 @@ class EthService(
     )
   }
 
+  @RpcMethod("txpool_contentFrom")
+  def txpoolContent(from: String): TxPoolContent = applyOnAccountView { nodeView =>
```

Comment:
we can use `Address` type directly.
In case the input is invalid (by structure or length) it will throw exception during parsing

Reply:
`from` parameter type change to `Address`
I also added in the python test an rpc call to `txpool_contentFrom` with an invalid input to check the exception expected